package.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "leaderboard-api",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "private": true,
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "build": "webpack"
+    },
```

Comment:
- [ ] Please also add a start script to your scripts just like you have added build and test script you can add `"start": "webpack --config webpack.config.js --watch"` this will help in staring the app with npm run start or npm start and will also enable webpack watch which will watch for changes in real time, which is really helpful in development environment :+1:

Reply:
Hello @PinkMoon25, Thanks for your feedback. I got the issues   👍